challenges/prototypes.js

```diff
@@ -20,7 +20,7 @@ function CuboidMaker(options) {
 CuboidMaker.prototype.volume = function volume() {
   return  `${this.length * this.width * this.height}`;
```

Comment:
you can also use an anonymous function here like this: CuboidMaker.prototype.volume = function(){ return `volume: ${this.length * this.width * this.height}`;} (note that I added the string volume: before the variable to make use of the template string that you are using with the backticks ) also note that if you are not wanting to add in a string in the return that you can omit the use of the template string and just return the variables math operation like this: CuboidMaker.prototype.volume = function(){ return this.length * this.width * this.height;}